1.13/base/Dockerfile

```diff
@@ -13,6 +16,10 @@ RUN \
             gnupg \
             libsnmp-dev \
             make \
+        && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+        && echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list \
```

Comment:
Put these at the top above the first `apt-get update`, then you can drop the next two commands and add it to the base list.

Reply:
So these lines require curl and gnupg, which is why we need to install the base packages above first...